DAQ_System/Source/Adapter/LinearPotentiometer/linear_potentiometer.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+* Linear Potentiometer
+* Author:        Manuel DJC    
+* Modified By:   Cristian Cruz
+*          
+* Email:         Manuel.DeJesusContreras@UTDallas.edu
+*                cris14.alex@gmail.com
+* 
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#ifndef LINEAR_POTENTOMETER_H
+#define LINEAR_POTENTOMETER_H
+
+#include "mbed.h"
+
+#include "Adapter/Interfaces/ilinear_potentiometer.hpp"
+
+namespace adapter {
+
+class LinearPotentiometer : public adapter::ILinear_Potentiometer {
+    public:
+        LinearPotentiometer(PinName p);
+        virtual ~LinearPotentiometer();
+        void setPin(PinName p);
+        double getLast();
```

Comment:
Perhaps rename `getLast()` to `GetLatestDisplacement()`
Pin names won't be re-assigned, as we can simply initialize a `LinearPotentiometer` object per pin